example/src/main/scala/example/PlainTextBenchmarkServer.scala

```diff
@@ -33,7 +33,9 @@ object Main extends App {
   private def server(response: Response[Any, Nothing]) =
     Server.app(app(response)) ++
       Server.port(8080) ++
+      Server.error(_ => UIO.unit) ++
       Server.keepAlive ++
-      Server.disableLeakDetection ++ Server.consolidateFlush
+      Server.disableLeakDetection ++
+      Server.consolidateFlush
```

Comment:
Example doesn't need changing.

Reply:
This is for benchmark @tusharmath, this will make benchmark for forks robust. Also we wanted to check there is no performance penalty.

Reply:
Changed to be used during benchmarking phase

Reply:
I would be interested in knowning how #606 will fix it.